lua/efmls-configs/linters/codespell.lua

```diff
@@ -0,0 +1,19 @@
+-- Metadata
+-- languages: misc
+-- url: https://github.com/codespell-project/codespell
+
+local fs = require("efmls-configs.fs")
+
+local linter = "codespell"
+
+-- HACK piping to `sed` to change severity to warning
+local command = string.format("%s ${INPUT} | sed 's/^/w /'", fs.executable(linter))
+
+return {
+	prefix = linter,
+	lintSource = linter,
+	lintCommand = command,
+	lintIgnoreExitCode = true,
+	lintStdin = false,
+	lintFormats = { "%t %f:%l:%m" },
```

Comment:
Since we won't need `sed` you can then change this to `%f:%l %m` to get proper the messages